modules/kubernetes-addons/main.tf

```diff
@@ -122,6 +122,14 @@ module "cert_manager" {
   addon_context     = local.addon_context
 }
 
+module "aws_pca_issuer" {
+  count             = var.enable_aws_pca_issuer ? 1 : 0
+  source            = "./aws-pca-issuer"
+  helm_config       = var.aws_pca_issuer_helm_config
+  manage_via_gitops = var.argocd_manage_add_ons
+  addon_context     = local.addon_context
```

Comment:
need to pass `irsa_iam_permissions_boundary`

Reply:
role_path and permission_boundary is moving to `addon_context` in [PR](https://github.com/aws-samples/aws-eks-accelerator-for-terraform/pull/351)